pkgs/applications/video/avxsynth/default.nix

```diff
@@ -1,26 +1,42 @@
-{ stdenv, fetchurl, ffmpeg, autoconf, automake, libtool, pkgconfig, log4cpp
-, pango, cairo, python, libjpeg, ffms
-, enableQt ? true, qt4}:
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig
+, cairo, ffmpeg, ffms, libjpeg, log4cpp, pango
+, avxeditSupport ? false, qt4 ? null
+}:
+
+let
+  inherit (stdenv.lib) enableFeature optional;
+in
 
 stdenv.mkDerivation rec {
-  name = "avxsynth-4.0-e153e672bf";
+  name = "avxsynth-${version}";
+  version = "2015-4-7";
```

Comment:
Yeah, I wasn't sure but I'll update it, might need to add something to make it clearer in #6539